cinema_service/settings.py

```
@@ -20,12 +25,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ["SECRET_KEY"]
```

Accessing SECRET_KEY with `os.environ["SECRET_KEY"]` will raise a KeyError if the variable is not set. If you expect the container to fail loudly in production that's acceptable, but for smoother development you may prefer `os.environ.get("SECRET_KEY")` with a clear default or custom error message. At minimum document that SECRET_KEY must be provided via env in production (requirement 3.2).